packages/target-web3-v1/src/codegen/events.ts

```diff
@@ -73,7 +73,7 @@ function codegenEventDeclaration(event: EventDeclaration, overloadedName?: strin
 
 function codegenOutputTypesForEvents(outputs: EventArgDeclaration[]): string {
   return `{
-    ${outputs.map((t) => t.name && `${t.name}: ${codegenOutputType(t.type)}, `).join('')}
-    ${outputs.map((t, i) => `${i}: ${codegenOutputType(t.type)}`).join(', ')}
+    ${outputs.map((t) => t.name && `${t.name}: ${codegenOutputType({ useStructs: true }, t.type)}, `).join('')}
+    ${outputs.map((t, i) => `${i}: ${codegenOutputType({ useStructs: true }, t.type)}`).join(', ')}
```

Comment:
any chance we can reverse this function's type params to `codegenOutputType(type, options)` instead of `codegenOutputType(options, type)`? The first is more likely to not break code, since it doesn't change the position of the type param.

Reply:
Seems like most codegen functions that take in an options param also take it as the last parameter